openspec/changes/lambda-tailscale-exit-node/design.md

```
@@ -0,0 +1,116 @@
+      "src": ["tag:lambda"],
+      "dst": ["autogroup:internet:*"]
+    }
+  ]
+}
+```
+
+**Rationale (from security review):** Without explicit ACLs, a compromised Lambda could access every device on the tailnet (including the Raspberry Pi). Restrict to internet-only traffic through the exit node.
+
+## Architecture diagram
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)